sensor_msgs/LaserScan[] group_laser